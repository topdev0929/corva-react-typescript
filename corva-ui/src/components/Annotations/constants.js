export const SET_ANNOTATIONS = 'SET_ANNOTATIONS';
export const APPEND_ANNOTATIONS = 'APPEND_ANNOTATIONS';
export const ADD_ANNOTATION = 'ADD_ANNOTATION';
export const DELETE_ANNOTATION = 'DELETE_ANNOTATION';
export const EDIT_ANNOTATION = 'EDIT_ANNOTATION';

export const ANNOTATION_ACTIVE_PERIODS = {
  '24h': { label: 'Next 24 hrs', time: { amount: 24, key: 'h' } },
  '12h': { label: 'Next 12 hrs', time: { amount: 12, key: 'h' } },
  '48h': { label: 'Next 48 hrs', time: { amount: 48, key: 'h' } },
  forever: { label: 'Forever' },
};
