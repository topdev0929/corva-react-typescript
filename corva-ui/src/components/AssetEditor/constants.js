export const PER_PAGE = 200;

// The distance in pixels before the end of the items that will trigger a call to loadMore.
export const SCROLL_THRESHOLD = 400;
