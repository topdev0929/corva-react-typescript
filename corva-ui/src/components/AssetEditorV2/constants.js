export const PER_PAGE = 200;

// The distance in pixels before the end of the items that will trigger a call to loadMore.
export const SCROLL_THRESHOLD = 400;

export const ACTIVE_STATUS = 'active';

export const ACTIVE_ASSET_ID = 'activeAsset';

export const ACTIVE_ASSETS_ID = 'activeAssets';
