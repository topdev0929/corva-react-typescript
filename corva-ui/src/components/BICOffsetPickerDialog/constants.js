export const DEFAULT_WELL_SECTIONS = [
  { name: 'Best Wells', wells: [], expanded: false },
  { name: 'Best Surface Wells', wells: [], expanded: false },
  { name: 'Best Intermediate Wells', wells: [], expanded: false },
  { name: 'Best Production Vertical Wells', wells: [], expanded: false },
  { name: 'Best Production Lateral Wells', wells: [], expanded: false },
  { name: 'Best Production Curve Wells', wells: [], expanded: false },
];
