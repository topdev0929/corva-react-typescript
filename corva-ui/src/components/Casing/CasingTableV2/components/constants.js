export const CONTAINER_XS = 12;

export const OD_ID_DEP_COLUMNS = [
  {
    label: 'OD',
    key: 'outer_diameter',
    unitType: 'shortLength',
    width: 80,
  },
  {
    label: 'ID',
    key: 'inner_diameter',
    unitType: 'shortLength',
    width: 80,
  },
  {
    label: 'Adjusted Linear Weight',
    key: 'linear_weight',
    unitType: 'massPerLength',
    width: 120,
  },
  {
    label: '# of Joints',
    key: 'number_of_joints',
    width: 80,
  },
  {
    label: 'Component Length',
    key: 'component_length',
    unitType: 'length',
    width: 80,
  },
];
