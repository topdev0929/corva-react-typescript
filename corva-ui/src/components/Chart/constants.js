export const MAX_ZOOM = 10;

export const ZOOM_TYPE = {
  zoomIn: 'zoom in',
  zoomOut: 'zoom out',
};

export const HOT_KEYS = {
  arrowUp: 38,
  arrowDown: 40,
};
export const MIN_CHART_HEIGHT = 182;

export const X_AXIS_RIGHT_GAP = 46;
