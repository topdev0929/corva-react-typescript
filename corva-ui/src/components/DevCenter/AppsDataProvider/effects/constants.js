export const APPS_DATA_REQUESTS = {
  WELLS: 'WELLS',
  RIGS: 'RIGS',
  FRAC_FLEETS: 'FRAC_FLEETS',
  FRAC_FLEET_WELLS: 'FRAC_FLEET_WELLS',
  PAD_WELLS: 'PAD_WELLS',
};
