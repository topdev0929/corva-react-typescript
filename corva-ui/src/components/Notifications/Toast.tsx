import { cloneElement, useEffect } from 'react';
import classNames from 'classnames';
import { noop } from 'lodash';

import { SnackbarContent, Typography, Grow, Tooltip } from '@material-ui/core';

import {
  CheckCircleOutline as CheckCircleOutlineIcon,
  ReportProblemOutlined as ReportProblemOutlinedIcon,
  ReportOutlined as ReportOutlinedIcon,
  InfoOutlined as InfoOutlinedIcon,
  Close as CloseIcon,
  Check as CheckIcon,
} from '@material-ui/icons';

import IconButton from '~/components/IconButton';
import { NOTIFICATION_VARIANTS } from '~/constants/notifications';

import { Notification } from './types';

import { useStyles } from './Toast.styles';

export const PAGE_NAME = 'NotificationPopup';

const icons = {
  [NOTIFICATION_VARIANTS.success]: CheckCircleOutlineIcon,
  [NOTIFICATION_VARIANTS.warning]: ReportProblemOutlinedIcon,
  [NOTIFICATION_VARIANTS.error]: ReportOutlinedIcon,
  [NOTIFICATION_VARIANTS.info]: InfoOutlinedIcon,
  [NOTIFICATION_VARIANTS.neutral]: CheckIcon,
};

interface ToastProps {
  onDismissClick: (id: string) => void;
  toast: Notification;
}

export const Toast = ({ onDismissClick, toast, ...other }: ToastProps): JSX.Element => {
  const {
    alertLevel,
    autoHideDuration = 3000,
    content,
    hideIcon,
    icon,
    id,
    onClick = noop,
    onClose = noop,
    timestamp,
    title,
    tooltipProps,
    variant = 'info',
  } = toast;

  useEffect(() => {
    const timeout = setTimeout(() => onDismissClick(id), autoHideDuration);

    return () => clearTimeout(timeout);
  }, []);

  const styles = useStyles();

  const Icon = icons[variant];

  return (
    <Grow appear in>
      <SnackbarContent
        classes={{
          root: classNames(
            styles.main,
            !alertLevel && styles[variant],
            alertLevel && styles.alertNotification,
            alertLevel && styles[`alertLevels-${alertLevel}`]
          ),
          action: styles.actionContainer,
          message: styles.messageContainer,
        }}
        aria-describedby="client-snackbar"
        message={
          <Tooltip
            {...tooltipProps}
            title={content}
            classes={{ tooltip: styles.tooltip }}
            placement="bottom"
          >
            <div
              id="client-snackbar"
              className={styles.messageWrapper}
              onClick={onClick}
              data-testid={`${PAGE_NAME}_status_${NOTIFICATION_VARIANTS[variant]}`}
            >
              {!hideIcon && (
                <div className={styles.iconWrapper}>
                  {icon ? (
                    cloneElement(icon, {
                      className: styles.icon,
                      nativeColor: '#fff',
                    })
                  ) : (
                    <Icon htmlColor="#fff" className={styles.icon} />
                  )}
                </div>
              )}
              <div className={styles.message}>
                <div className={styles.messageTop}>
                  {title && (
                    <Typography
                      data-testid={`${PAGE_NAME}_title`}
                      variant="subtitle2"
                      className={classNames(styles.title, { [styles.alertTitle]: alertLevel })}
                    >
                      {title}
                    </Typography>
                  )}
                  {timestamp && (
                    <Typography
                      className={classNames(styles.timestamp, !alertLevel && styles[variant])}
                    >
                      {timestamp}
                    </Typography>
                  )}
                </div>
                <div className={classNames(styles.messageContent, !alertLevel && styles[variant])}>
                  <Typography
                    data-testid={`${PAGE_NAME}_content`}
                    className={classNames(alertLevel ? styles.alertContent : styles.content)}
                  >
                    {content}
                  </Typography>
                </div>
              </div>
            </div>
          </Tooltip>
        }
        action={[
          <IconButton
            data-testid={`${PAGE_NAME}_closeButton`}
            key="close"
            aria-label="Close"
            color="inherit"
            onClick={() => {
              onClose();
              onDismissClick(id);
            }}
            size="small"
            // @ts-ignore
            tooltipProps={{ title: 'Close', classes: { tooltip: styles.tooltip } }}
            className={styles.closeIconButton}
          >
            <CloseIcon className={styles.closeIcon} htmlColor={alertLevel ? '#000' : '#fff'} />
          </IconButton>,
        ]}
        {...other}
      />
    </Grow>
  );
};
