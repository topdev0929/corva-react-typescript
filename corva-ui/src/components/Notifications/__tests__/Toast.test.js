import { render } from '@testing-library/react';

import { NOTIFICATION_VARIANTS } from '~/constants/notifications';

import { Toast } from '../Toast';

const withProps = (extraToastProps = {}) => ({
  toast: {
    content: 'Toast content',
    id: 42,
    ...extraToastProps,
  },
  onDismissClick: () => {},
});

describe('Toast', () => {
  describe('snapshots', () => {
    it('matches default snapshot', () => {
      const container = render(<Toast {...withProps()} />).baseElement;
      expect(container).toMatchSnapshot();
    });

    const testVariant = variant =>
      it(`matches ${variant} snapshot`, () => {
        const container = render(<Toast {...withProps({ variant })} />).baseElement;
        expect(container).toMatchSnapshot();
      });

    Object.values(NOTIFICATION_VARIANTS).forEach(testVariant);
  });
});
