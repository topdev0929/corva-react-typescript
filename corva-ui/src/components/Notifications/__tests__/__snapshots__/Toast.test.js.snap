// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Toast snapshots matches default snapshot 1`] = `
<body>
  <div>
    <div
      aria-describedby="client-snackbar"
      class="MuiPaper-root MuiSnackbarContent-root makeStyles-main-1 makeStyles-info-5 MuiPaper-elevation6"
      role="alert"
      style="opacity: 1; transform: scale(1, 1); transition: opacity 0ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,transform 0ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
    >
      <div
        class="MuiSnackbarContent-message makeStyles-messageContainer-15"
      >
        <div
          class="makeStyles-messageWrapper-16"
          data-testid="NotificationPopup_status_info"
          id="client-snackbar"
          title="Toast content"
        >
          <div
            class="makeStyles-iconWrapper-11"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root makeStyles-icon-12"
              color="#fff"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M11 7h2v2h-2zm0 4h2v6h-2zm1-9C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8z"
              />
            </svg>
          </div>
          <div
            class="makeStyles-message-17"
          >
            <div
              class="makeStyles-messageTop-18"
            />
            <div
              class="makeStyles-messageContent-19 makeStyles-info-5"
            >
              <p
                class="MuiTypography-root makeStyles-content-9 MuiTypography-body1"
                data-testid="NotificationPopup_content"
              >
                Toast content
              </p>
            </div>
          </div>
        </div>
      </div>
      <div
        class="MuiSnackbarContent-action makeStyles-actionContainer-21"
      >
        <button
          aria-label="Close"
          class="MuiButtonBase-root MuiIconButton-root makeStyles-root-27 makeStyles-closeIconButton-13 MuiIconButton-colorInherit MuiIconButton-sizeSmall makeStyles-sizeSmall-29"
          data-testid="NotificationPopup_closeButton"
          tabindex="0"
          title="Close"
          type="button"
          variant="default"
        >
          <span
            class="MuiIconButton-label"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root makeStyles-closeIcon-14"
              color="#fff"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
              />
            </svg>
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </button>
      </div>
    </div>
  </div>
</body>
`;

exports[`Toast snapshots matches error snapshot 1`] = `
<body>
  <div>
    <div
      aria-describedby="client-snackbar"
      class="MuiPaper-root MuiSnackbarContent-root makeStyles-main-137 makeStyles-error-139 MuiPaper-elevation6"
      role="alert"
      style="opacity: 1; transform: scale(1, 1); transition: opacity 0ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,transform 0ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
    >
      <div
        class="MuiSnackbarContent-message makeStyles-messageContainer-151"
      >
        <div
          class="makeStyles-messageWrapper-152"
          data-testid="NotificationPopup_status_error"
          id="client-snackbar"
          title="Toast content"
        >
          <div
            class="makeStyles-iconWrapper-147"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root makeStyles-icon-148"
              color="#fff"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M15.73 3H8.27L3 8.27v7.46L8.27 21h7.46L21 15.73V8.27L15.73 3zM19 14.9L14.9 19H9.1L5 14.9V9.1L9.1 5h5.8L19 9.1v5.8z"
              />
              <circle
                cx="12"
                cy="16"
                r="1"
              />
              <path
                d="M11 7h2v7h-2z"
              />
            </svg>
          </div>
          <div
            class="makeStyles-message-153"
          >
            <div
              class="makeStyles-messageTop-154"
            />
            <div
              class="makeStyles-messageContent-155 makeStyles-error-139"
            >
              <p
                class="MuiTypography-root makeStyles-content-145 MuiTypography-body1"
                data-testid="NotificationPopup_content"
              >
                Toast content
              </p>
            </div>
          </div>
        </div>
      </div>
      <div
        class="MuiSnackbarContent-action makeStyles-actionContainer-157"
      >
        <button
          aria-label="Close"
          class="MuiButtonBase-root MuiIconButton-root makeStyles-root-163 makeStyles-closeIconButton-149 MuiIconButton-colorInherit MuiIconButton-sizeSmall makeStyles-sizeSmall-165"
          data-testid="NotificationPopup_closeButton"
          tabindex="0"
          title="Close"
          type="button"
          variant="default"
        >
          <span
            class="MuiIconButton-label"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root makeStyles-closeIcon-150"
              color="#fff"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
              />
            </svg>
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </button>
      </div>
    </div>
  </div>
</body>
`;

exports[`Toast snapshots matches info snapshot 1`] = `
<body>
  <div>
    <div
      aria-describedby="client-snackbar"
      class="MuiPaper-root MuiSnackbarContent-root makeStyles-main-69 makeStyles-info-73 MuiPaper-elevation6"
      role="alert"
      style="opacity: 1; transform: scale(1, 1); transition: opacity 0ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,transform 0ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
    >
      <div
        class="MuiSnackbarContent-message makeStyles-messageContainer-83"
      >
        <div
          class="makeStyles-messageWrapper-84"
          data-testid="NotificationPopup_status_info"
          id="client-snackbar"
          title="Toast content"
        >
          <div
            class="makeStyles-iconWrapper-79"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root makeStyles-icon-80"
              color="#fff"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M11 7h2v2h-2zm0 4h2v6h-2zm1-9C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.41 0-8-3.59-8-8s3.59-8 8-8 8 3.59 8 8-3.59 8-8 8z"
              />
            </svg>
          </div>
          <div
            class="makeStyles-message-85"
          >
            <div
              class="makeStyles-messageTop-86"
            />
            <div
              class="makeStyles-messageContent-87 makeStyles-info-73"
            >
              <p
                class="MuiTypography-root makeStyles-content-77 MuiTypography-body1"
                data-testid="NotificationPopup_content"
              >
                Toast content
              </p>
            </div>
          </div>
        </div>
      </div>
      <div
        class="MuiSnackbarContent-action makeStyles-actionContainer-89"
      >
        <button
          aria-label="Close"
          class="MuiButtonBase-root MuiIconButton-root makeStyles-root-95 makeStyles-closeIconButton-81 MuiIconButton-colorInherit MuiIconButton-sizeSmall makeStyles-sizeSmall-97"
          data-testid="NotificationPopup_closeButton"
          tabindex="0"
          title="Close"
          type="button"
          variant="default"
        >
          <span
            class="MuiIconButton-label"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root makeStyles-closeIcon-82"
              color="#fff"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
              />
            </svg>
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </button>
      </div>
    </div>
  </div>
</body>
`;

exports[`Toast snapshots matches neutral snapshot 1`] = `
<body>
  <div>
    <div
      aria-describedby="client-snackbar"
      class="MuiPaper-root MuiSnackbarContent-root makeStyles-main-171 makeStyles-neutral-176 MuiPaper-elevation6"
      role="alert"
      style="opacity: 1; transform: scale(1, 1); transition: opacity 0ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,transform 0ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
    >
      <div
        class="MuiSnackbarContent-message makeStyles-messageContainer-185"
      >
        <div
          class="makeStyles-messageWrapper-186"
          data-testid="NotificationPopup_status_neutral"
          id="client-snackbar"
          title="Toast content"
        >
          <div
            class="makeStyles-iconWrapper-181"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root makeStyles-icon-182"
              color="#fff"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M9 16.17L4.83 12l-1.42 1.41L9 19 21 7l-1.41-1.41z"
              />
            </svg>
          </div>
          <div
            class="makeStyles-message-187"
          >
            <div
              class="makeStyles-messageTop-188"
            />
            <div
              class="makeStyles-messageContent-189 makeStyles-neutral-176"
            >
              <p
                class="MuiTypography-root makeStyles-content-179 MuiTypography-body1"
                data-testid="NotificationPopup_content"
              >
                Toast content
              </p>
            </div>
          </div>
        </div>
      </div>
      <div
        class="MuiSnackbarContent-action makeStyles-actionContainer-191"
      >
        <button
          aria-label="Close"
          class="MuiButtonBase-root MuiIconButton-root makeStyles-root-197 makeStyles-closeIconButton-183 MuiIconButton-colorInherit MuiIconButton-sizeSmall makeStyles-sizeSmall-199"
          data-testid="NotificationPopup_closeButton"
          tabindex="0"
          title="Close"
          type="button"
          variant="default"
        >
          <span
            class="MuiIconButton-label"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root makeStyles-closeIcon-184"
              color="#fff"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
              />
            </svg>
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </button>
      </div>
    </div>
  </div>
</body>
`;

exports[`Toast snapshots matches success snapshot 1`] = `
<body>
  <div>
    <div
      aria-describedby="client-snackbar"
      class="MuiPaper-root MuiSnackbarContent-root makeStyles-main-35 makeStyles-success-36 MuiPaper-elevation6"
      role="alert"
      style="opacity: 1; transform: scale(1, 1); transition: opacity 0ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,transform 0ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
    >
      <div
        class="MuiSnackbarContent-message makeStyles-messageContainer-49"
      >
        <div
          class="makeStyles-messageWrapper-50"
          data-testid="NotificationPopup_status_success"
          id="client-snackbar"
          title="Toast content"
        >
          <div
            class="makeStyles-iconWrapper-45"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root makeStyles-icon-46"
              color="#fff"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M16.59 7.58L10 14.17l-3.59-3.58L5 12l5 5 8-8zM12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 18c-4.42 0-8-3.58-8-8s3.58-8 8-8 8 3.58 8 8-3.58 8-8 8z"
              />
            </svg>
          </div>
          <div
            class="makeStyles-message-51"
          >
            <div
              class="makeStyles-messageTop-52"
            />
            <div
              class="makeStyles-messageContent-53 makeStyles-success-36"
            >
              <p
                class="MuiTypography-root makeStyles-content-43 MuiTypography-body1"
                data-testid="NotificationPopup_content"
              >
                Toast content
              </p>
            </div>
          </div>
        </div>
      </div>
      <div
        class="MuiSnackbarContent-action makeStyles-actionContainer-55"
      >
        <button
          aria-label="Close"
          class="MuiButtonBase-root MuiIconButton-root makeStyles-root-61 makeStyles-closeIconButton-47 MuiIconButton-colorInherit MuiIconButton-sizeSmall makeStyles-sizeSmall-63"
          data-testid="NotificationPopup_closeButton"
          tabindex="0"
          title="Close"
          type="button"
          variant="default"
        >
          <span
            class="MuiIconButton-label"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root makeStyles-closeIcon-48"
              color="#fff"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
              />
            </svg>
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </button>
      </div>
    </div>
  </div>
</body>
`;

exports[`Toast snapshots matches warning snapshot 1`] = `
<body>
  <div>
    <div
      aria-describedby="client-snackbar"
      class="MuiPaper-root MuiSnackbarContent-root makeStyles-main-103 makeStyles-warning-106 MuiPaper-elevation6"
      role="alert"
      style="opacity: 1; transform: scale(1, 1); transition: opacity 0ms cubic-bezier(0.4, 0, 0.2, 1) 0ms,transform 0ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;"
    >
      <div
        class="MuiSnackbarContent-message makeStyles-messageContainer-117"
      >
        <div
          class="makeStyles-messageWrapper-118"
          data-testid="NotificationPopup_status_warning"
          id="client-snackbar"
          title="Toast content"
        >
          <div
            class="makeStyles-iconWrapper-113"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root makeStyles-icon-114"
              color="#fff"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M12 5.99L19.53 19H4.47L12 5.99M12 2L1 21h22L12 2zm1 14h-2v2h2v-2zm0-6h-2v4h2v-4z"
              />
            </svg>
          </div>
          <div
            class="makeStyles-message-119"
          >
            <div
              class="makeStyles-messageTop-120"
            />
            <div
              class="makeStyles-messageContent-121 makeStyles-warning-106"
            >
              <p
                class="MuiTypography-root makeStyles-content-111 MuiTypography-body1"
                data-testid="NotificationPopup_content"
              >
                Toast content
              </p>
            </div>
          </div>
        </div>
      </div>
      <div
        class="MuiSnackbarContent-action makeStyles-actionContainer-123"
      >
        <button
          aria-label="Close"
          class="MuiButtonBase-root MuiIconButton-root makeStyles-root-129 makeStyles-closeIconButton-115 MuiIconButton-colorInherit MuiIconButton-sizeSmall makeStyles-sizeSmall-131"
          data-testid="NotificationPopup_closeButton"
          tabindex="0"
          title="Close"
          type="button"
          variant="default"
        >
          <span
            class="MuiIconButton-label"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root makeStyles-closeIcon-116"
              color="#fff"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
              />
            </svg>
          </span>
          <span
            class="MuiTouchRipple-root"
          />
        </button>
      </div>
    </div>
  </div>
</body>
`;
